méli-mélo/th-empathy/empathy.html

```diff
@@ -0,0 +1,93 @@
+---
+title: Empathy theme
+dateModified: 2024-04-30
+description: Empathy CSS styles
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- style.css
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.css
+script:
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.js
+layout: without-h1
+---
+<h1 property="name" id="wb-cont" dir="ltr" class="empathy">{{ page.title }}</h1>
+{% assign metadata = site.pages | where: "output", "false" | where: "componentName", "th-empathy" | first %}
+<p>Sponsor: {{ metadata.sponsor }}</p>
```

Comment:
We are missing one small paragraph describing this special thématique. A small description like "This thématique is to emphasis the empathy feeling regarding the web content presented toward the client. Like during a lost of a love one life event."

This is needed to satisfy at the requirement of defining the purpose for a thématique.